azure-pipelines.yml

```diff
@@ -2,10 +2,15 @@ trigger:
   branches:
     include:
     - master
-pr: none
+pr:
+- master
+
+variables:
+  BuildConfiguration: Release
 
 jobs:
 - job: scrape_x64
+  condition: false
```

Comment:
I'll revert these `condition: false` lines. They were only added to save Azure Pipelines resources while I iterated on these changes.